### PR TITLE
Added contextual message

### PR DESCRIPTION
**Pull request by jvt**
Context messages are slightly smaller than the "x / 4 pull requests submitted" string.

### CODE REVIEW

File: views/index.hbs [hunk 1]
```diff
@@ -36,7 +36,9 @@
         {{#if prs}}
             <div class="row">
                 <div class="large-12 columns">
-                    <h2><strong>{{ prs.length }} / 4</strong> pull requests submitted in Hacktober</h2>
+                    <h2><strong>{{ prs.length }} / 4</strong> pull requests submitted in Hacktober.</h2>
+                    <br>
+                    <h3 class="center"><strong>{{ statement }}</strong></h3>
```
Comment (jenkoian) [inline views/index.hbs]: Can get rid of the class here as not used.

File: views/index.hbs [hunk 2]
```diff
@@ -36,7 +36,9 @@
         {{#if prs}}
             <div class="row">
                 <div class="large-12 columns">
-                    <h2><strong>{{ prs.length }} / 4</strong> pull requests submitted in Hacktober</h2>
+                    <h2><strong>{{ prs.length }} / 4</strong> pull requests submitted in Hacktober.</h2>
```
Comment (jenkoian) [inline views/index.hbs]: I'm thinking we get rid of the 'pull requests submitted in Hacktober' bit now, what you think?

File: views/index.hbs (hunk 1 above)
Comment (jenkoian) [inline views/index.hbs]: Can you also add this to the line below line 69 as well (I know the logic here isn't perfect, but from what I can tell you can't do much in terms of expressions in handlebars and didn't bother adding a custom helper or anything)

File: views/index.hbs (hunk 2 above)
Comment (jvt) [inline views/index.hbs]: I think that would make the most sense to do.

File: views/index.hbs [hunk 3]
```diff
@@ -36,7 +36,9 @@
         {{#if prs}}
             <div class="row">
                 <div class="large-12 columns">
-                    <h2><strong>{{ prs.length }} / 4</strong> pull requests submitted in Hacktober</h2>
+                    <h2><strong>{{ prs.length }} / 4</strong></h2>
+                    <br>
+                    <h3><strong>{{ statement }}</strong></h3>
```
Comment (jenkoian) [inline views/index.hbs]: Last thing, can you copy this line to just below line 69 too. Cheers!

**Comment by jenkoian** [comment]
Thanks!

**Pull request merged**